src/features/donations/components/ShareOptions.jsx

```diff
@@ -92,7 +93,7 @@ const ShareOptions = (props) => {
         className={styles.shareIcon}
         onClick={() =>
           shareClicked(
-            `https://www.facebook.com/sharer.php?u=${urlToShare}&quote=${textToShare}`,
+            `https://www.facebook.com/sharer.php?u=${urlToShare}&quote=${textToShare}&hashtag=%23StopTalkingStartPlanting`,
```

Comment:
We already have another resource next to `donate:textToShareLinkedin` named `donate:textToShare` containing some tags (but also the Twitter user name) at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/92fc7b0e60e32a7fbefdac4e7ba6b231740dbee2/public/static/locales/en/donate.json#L64